src/System.Private.CoreLib/shared/System/Text/UnicodeEncoding.cs

```diff
@@ -29,10 +29,10 @@ public class UnicodeEncoding : Encoding
         private static readonly byte[] s_bigEndianPreamble = new byte[2] { 0xfe, 0xff };
         private static readonly byte[] s_littleEndianPreamble = new byte[2] { 0xff, 0xfe };
 
-        private bool isThrowException = false;
+        private readonly bool isThrowException;
```

Comment:
If you would like, feel free to rename these fields to start with `_` to follow the corefx coding conventions while you are on it so that it looks the same as the other encoding impls.

Reply:
Because the other have extra `base(bigEndian ? 1201 : 1200)  //Set the data item.` logic and I was not sure how you'd like copying that